### ISSUE TITLE
Help - Batch add data to Table Storage

### ISSUE THREAD

**Issue by LeoniSantos** [azurefieldnotes/Avanade.AzureStorage#2]
Hello cspeers,

It is not an issue, I just didn't know where to post, I am trying to batch include data into Azure Table Storage, based on documentation we can write up to 100 rows by time, but I have failed into performing it using REST calls, I saw you have developed this full module and thought you may help, by sharing a direction or want to know if you have done it before, have an example code.

basically I am using Azure runbook to retrieve o365 data and want to write it to azure table in better performance, include row by row is taking too long.

thanks a lot
